ui/app/controllers/clients/client/index.js

```diff
@@ -211,6 +211,7 @@ export default class ClientController extends Controller.extend(
       { key: 'complete', label: 'Complete' },
       { key: 'failed', label: 'Failed' },
       { key: 'lost', label: 'Lost' },
+      { key: 'unknown', label: 'Unknown' },
```

Comment:
This change (and similar changes to jobs/job/allocations controller, etc.) all correspond to these filters:

![image](https://user-images.githubusercontent.com/713991/162802772-58538ed8-5263-44f8-bf32-3920f5337350.png)